__tests__/cookie-banner.test.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-env jest */
+
+const { setupPage } = require('../lib/jest-utilities.js')
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.testPort
+
+let page
+let baseUrl = 'http://localhost:' + PORT
+
+beforeEach(async () => {
+  page = await setupPage()
+})
+
+afterEach(async () => {
+  await page.deleteCookie({ name: 'design_system_cookies_policy' })
+  await page.close()
+})
+
+const COOKIE_BANNER_SELECTOR = '[data-module="govuk-cookie-banner"]'
+
+describe('Cookie banner', () => {
+  describe('is hidden', () => {
```

Comment:
Not too sure about the way the 'thing we're asserting' ends up in the describe block's name here. We normally use `describe` blocks to group tests that cover a certain set of components or a particular context (like JS being enabled, or disabled) but here we're sort of grouping them by outcome?